test/workspaces/pipfile-dev-deps-only/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
Customer's problem occurred when `Pipfile` did not contain `packages` at all. Worth to test this scenario as well